src/Halcyon/Datasource/FileDatasource.php

```diff
@@ -348,54 +299,38 @@ protected function makeDirectoryPath($dirName, $relativePath = '')
     }
 
     /**
-     * Helper to make file path.
+     * Helper method to make the full file path to the model.
      *
-     * @param  string  $dirName
-     * @param  string  $fileName
-     * @param  string  $extension
-     * @return string
+     * @param string $dirName The directory in which the model is stored.
+     * @param string $fileName The filename of the model.
+     * @param string $extension The file extension of the model.
+     * @return string The full file path.
      */
-    protected function makeFilePath(string $dirName, string $fileName, string $extension)
+    protected function makeFilePath(string $dirName, string $fileName, string $extension): string
     {
         return $this->makeDirectoryPath($dirName, $fileName . '.' . $extension);
     }
 
-    /**
-     * Generate a cache key unique to this datasource.
-     *
-     * @param  string  $name
-     * @return string
-     */
-    public function makeCacheKey($name = '')
-    {
-        return crc32($this->basePath . $name);
-    }
```

Comment:
Why was this method removed?

Reply:
Same reason as this: https://github.com/wintercms/storm/pull/64/files/c99be36471e01ac8ef93e5c0258a514ac850b866#r771805382